packages/circuit-ui/index.ts

```diff
@@ -126,6 +126,7 @@ export type {
   TabPanelProps,
   TabProps,
 } from './components/Tabs/index.js';
+export { SkipLink } from './components/SkipLink/index.js';
```

Comment:
I wonder whether we should also export `SkipLinkProps` as an alias of the `AnchorProps` 🤔